client/a8c-for-agencies/sections/overview/body/intro-cards/style.scss

```diff
@@ -0,0 +1,24 @@
+.intro-cards {
+	.dot-pager__page {
+		p,
+		li {
+			font-size: rem(16px);
+			font-weight: 400;
+			line-height: 24px;
+			color: var(--studio-gray-80);
```

Comment:
Similar to this: https://github.com/Automattic/wp-calypso/pull/88733#discussion_r1533386721 suggestion we could replace `--studio` vars

Reply:
This is good to know – I've removed these `color` styles entirely as the default body `--color-text` variable is already equal to the `--studio-gray-80` value :+1: